tests/lib/rules/sort-imports.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * @fileoverview Tests for sort-imports rule.
+ * @author Christian Schuller
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var rule = require("../../../lib/rules/sort-imports"),
+    RuleTester = require("../../../lib/testers/rule-tester");
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+var ruleTester = new RuleTester(),
+    parserOptions = { ecmaVersion: 6, sourceType: "module" },
+    expectedError = {
+        message: "Imports should be sorted alphabetically.",
+        type: "ImportDeclaration"
+    },
+    ignoreCaseArgs = [ { ignoreCase: true } ];
+
+ruleTester.run("sort-imports", rule, {
+    valid: [
+        { code: "import * as B from './foo'; import A from './bar';", parserOptions: parserOptions },
```

Comment:
For readbility sake, can you please split the code into multiple lines? Something like this

```js
{ 
    code: 
        "import * as B from './foo';\n" + 
        "import A from './bar';", 
    parserOptions: parserOptions 
}
```